ethcore/db/src/db.rs

```diff
@@ -33,8 +33,6 @@ pub const COL_BODIES: Option<u32> = Some(2);
 pub const COL_EXTRA: Option<u32> = Some(3);
 /// Column for Traces
 pub const COL_TRACE: Option<u32> = Some(4);
-/// Column for the empty accounts bloom filter.
-pub const COL_ACCOUNT_BLOOM: Option<u32> = Some(5);
```

Comment:
When doing database migration, the data of this column should be deleted (not the column itself), so is good to keep the column, only for this purpose, I put the name `COL_ACOUNT_BLOOL_UNUSED`

Reply:
I was hesitant about backporting the migration as it could break 2.5 backwards compatibility, and if the accounts bloom savings are worth it (it's really tiny AFAIK).